addons/account/models/account_invoice.py

```diff
@@ -949,6 +949,13 @@ def _track_subtype(self, init_values):
             return 'account.mt_invoice_created'
         return super(AccountInvoice, self)._track_subtype(init_values)
 
+    @api.multi
+    def _get_groups(self):
+        self.ensure_one()
+        groups = self.env['account.tax.group']
+        for line in self.tax_line_ids:
+            groups |= line.tax_id.tax_group_id
+        return groups
```

Comment:
Ok I understand why you had to add the field and did that, you can use "formatLang" in python to format a float variable into a string containing the correct rounding and the symbol of the currency. That way you won't need to add a field which purpose is just to show the currency symbol.

from openerp.tools.misc import formatLang
amount = formatLang(self.env, your_float_variable, currency_obj=self.currency_id or self.company_id.currency_id)
and in the report you don't have to add the t-field-option

and voilà :)